File: app/src/main/java/br/com/wcisang/alojinha/service/BannerService.java
package br.com.wcisang.alojinha.service;

import br.com.wcisang.alojinha.service.response.BannerResponse;
import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by WCisang on 07/06/2018.
 */
public interface BannerService {

    @GET("/banner")
    Call<BannerResponse> getBannerList();
}

File: app/src/main/java/br/com/b2w/desafio/data/remote/BannerService.kt
package br.com.b2w.desafio.data.remote

import br.com.b2w.desafio.model.Banner
import br.com.b2w.desafio.model.response.LodjinhaResponse
import retrofit2.Call
import retrofit2.http.*

interface BannerService {
    @GET("banner")
    fun listar(
    ): Call<LodjinhaResponse<MutableList<Banner>>>
}